include/aws/http/private/connection_impl.h

```diff
@@ -52,6 +52,9 @@ struct aws_http_connection_vtable {
     void (*close)(struct aws_http_connection *connection);
     bool (*is_open)(const struct aws_http_connection *connection);
     void (*update_window)(struct aws_http_connection *connection, size_t increment_size);
+
+    /* HTTP/2 specific functions */
+    int (*change_settings)(struct aws_http_connection *connection, const struct aws_http2_change_settings_options *opt);
```

Comment:
We use *base approach instead of *impl... So from my understanding, this will be the "way".